hermes/src/api.rs

```diff
@@ -80,7 +74,8 @@ pub async fn run(store: Arc<Store>, mut update_rx: Receiver<()>, rpc_addr: Strin
         .route("/api/get_vaa_ccip", get(rest::get_vaa_ccip))
         .route("/api/price_feed_ids", get(rest::price_feed_ids))
         .with_state(state.clone())
-        .layer(CorsLayer::permissive()); // Permissive CORS layer to allow all origins
+        .layer(CorsLayer::permissive()) // Permissive CORS layer to allow all origins
+        .layer(Extension(QsQueryConfig::new(5, false))); // non-strict mode permits escaped [] in URL parameters
```

Comment:
i added this because the `ids[]` parameter gets escaped by OpenAPI. It seems like this is a common enough problem in HTTP clients that there's an option to permit it here, so i think it's worth enabling that.

Reply:
Can you document what is 5 and why it is chosen? It's the allowed depth i assume :?